addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -333,6 +349,20 @@ def _l10n_in_edi_ewaybill_post_invoice_edi(self, invoices):
             })
             inv_res = {"success": True, "attachment": attachment}
             res[invoices] = inv_res
+            body = Markup("""
+                <b>{}</b><br/>
+                <ul>
+                    <li>{}<i class="o-mail-Message-trackingSeparator fa fa-long-arrow-right mx-1 text-600"></i>{}</li>
+                    <li>{}<i class="o-mail-Message-trackingSeparator fa fa-long-arrow-right mx-1 text-600"></i>{}</li>
+                </ul>
+            """).format(
+                _('E-wayBill Sent'),
+                _('Number'),
+                str(response.get("data", False).get('ewayBillNo', 0)) or str(response.get("data", False).get('EwbNo', 0)),
```

Comment:
```suggestion
                str(response.get("data", {}).get('ewayBillNo', 0)) or str(response.get("data", {}).get('EwbNo', 0)),
```
Can not we suppose data is there if there is no error and just put response['data']